dome9/resource_dome9_cloudaccount_aws_test.go

```diff
@@ -253,10 +253,30 @@ net_sec {
       new_group_behavior = "ReadOnly"
       region             = "ap_northeast_3"
     }
-        regions {
+    regions {
       new_group_behavior = "ReadOnly"
       region             = "me_central_1"
     }
+    regions {
+      new_group_behavior = "ReadOnly"
+      region             = "ap_south_2"
+    }
+    regions {
+      new_group_behavior = "ReadOnly"
+      region             = "ap_southeast_3"
+    }
+    regions {
+      new_group_behavior = "ReadOnly"
+      region             = "ap_southeast_4"
+    }
+    regions {
+      new_group_behavior = "ReadOnly"
+      region             = "eu_central_2"
+    }
+    regions {
+      new_group_behavior = "ReadOnly"
+      region             = "eu_south_2"
```

Comment:
Please align with other region's fields positions

Reply:
Done (actually it was aligned, but some others above it weren't - now all are aligned properly)